<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APIAdaptDialog</class>
 <widget class="QDialog" name="APIAdaptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>应用APK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldAPI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openOldAPIFile">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="text">
    <string>旧版应用的信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldInvoke">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openOldInvokeFile">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="text">
    <string>新版应用的信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newInvoke">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openNewInvokeFile">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="executeAPIAdapt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始分析APK信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>分析结果</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newAPI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
